engine/table/src/main/java/io/deephaven/engine/table/impl/by/GroupByChunkedOperator.java

```diff
@@ -494,67 +478,69 @@ public void propagateInitialState(@NotNull final QueryTable resultTable) {
+                             ResettableWritableObjectChunk.makeResettableChunk();
+                     final ResettableWritableObjectChunk<RowSetBuilderRandom, Values> removedBuildersResettableChunk =
+                             ResettableWritableObjectChunk.makeResettableChunk();
+                     final RowSequence.Iterator destinationsIterator =
+                             stepDestinations.getRowSequenceIterator()) {
+
+                    // noinspection unchecked
+                    final WritableObjectChunk<WritableRowSet, Values> rowSetBackingChunk =
+                            rowSetResettableChunk.asWritableObjectChunk();
+                    // noinspection unchecked
+                    final WritableObjectChunk<RowSetBuilderRandom, Values> addedBuildersBackingChunk =
+                            addedBuildersResettableChunk.asWritableObjectChunk();
+                    // noinspection unchecked
+                    final WritableObjectChunk<RowSetBuilderRandom, Values> removedBuildersBackingChunk =
+                            removedBuildersResettableChunk.asWritableObjectChunk();
+
+                    while (destinationsIterator.hasMore()) {
+                        final long firstSliceDestination = destinationsIterator.peekNextKey();
+                        final long firstBackingChunkDestination =
+                                rowSets.resetWritableChunkToBackingStore(rowSetResettableChunk, firstSliceDestination);
+                        addedBuilders.resetWritableChunkToBackingStore(addedBuildersResettableChunk, firstSliceDestination);
+                        removedBuilders.resetWritableChunkToBackingStore(removedBuildersResettableChunk, firstSliceDestination);
+
+                        final long lastBackingChunkDestination =
+                                firstBackingChunkDestination + rowSetBackingChunk.size() - 1;
+                        final RowSequence initialDestinationsSlice =
+                                destinationsIterator.getNextRowSequenceThrough(lastBackingChunkDestination);
+
+                        initialDestinationsSlice.forAllRowKeys((final long destination) -> {
+                            final int backingChunkOffset =
+                                    Math.toIntExact(destination - firstBackingChunkDestination);
+                            WritableRowSet workingRowSet = rowSetBackingChunk.get(backingChunkOffset);
+                            if (workingRowSet == null) {
+                                // use the addRowSet as the new rowset
+                                final WritableRowSet addRowSet = nullToEmpty(extractAndClearBuilderRandom(addedBuildersBackingChunk, backingChunkOffset));
+                                rowSetBackingChunk.set(backingChunkOffset, live ? addRowSet.toTracking() : addRowSet);
                                 onReincarnated(destination);
```

Comment:
Can't be reincarnated, has to be brand new. New slots are handled by the helper, anyway.